crates/nu-std/testing.nu

```diff
@@ -155,8 +156,9 @@ export def ($test_function_name) [] {
     | str join (char lsep)
     | save $rendered_module_path
 
+    let plugins = $plugins | to json -r
     let result = (
-        ^$nu.current-exe --no-config-file -c $"use ($rendered_module_path) *; ($test_function_name)|to nuon"
+        ^$nu.current-exe --no-config-file --plugins $plugins -c $"use ($rendered_module_path) *; ($test_function_name)|to nuon"
```

Comment:
An empty list doesn't cause funny things here right?

Reply:
It did not in my testings, also it would just loop over nothing here: https://github.com/nushell/nushell/blob/d273ce89df3e801c4d8afbc99af11a71451478c5/src/main.rs#L415